<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintToolInterface</class>
 <widget class="QWidget" name="PaintToolInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>323</height>
   </rect>
  </property>
   <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="activeHLayout">
     <item>
      <widget class="QLabel" name="activeLabel">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="activeComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="maskHLayout">
     <item>
      <widget class="QLabel" name="maskLabel">
       <property name="text">
        <string>Mask:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="maskComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="eraseHLayout">
     <item>
      <spacer name="eraseHSpacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="eraseCheckBox">
       <property name="text">
        <string>Erase Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="eraseHSpacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="brushHLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="brushLabel">
       <property name="text">
        <string>Brush Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="brushHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="thresholdGroupBox">
     <property name="title">
      <string>Threshold</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="upperHLayout_top">
        <item>
         <widget class="QLabel" name="upperLabel">
          <property name="text">
           <string>Upper:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="upperHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="upperHLayout_bottom"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lowerHLayout_top">
        <item>
         <widget class="QLabel" name="lowerLabel">
          <property name="text">
           <string>Lower:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="lowerHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lowerHLayout_bottom"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
